tests/test_at.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import annotations
+
+from contextlib import contextmanager, suppress
+
+import numpy as np
+import pytest
+
+from array_api_compat import (
+    array_namespace,
+    at,
+    is_cupy_array,
+    is_dask_array,
+    is_jax_array,
+    is_torch_array,
+    is_torch_namespace,
+    is_pydata_sparse_array,
+    is_writeable_array,
+)
+from ._helpers import import_, all_libraries
+
+
+def assert_array_equal(a, b):
+    if is_pydata_sparse_array(a):
+        a = a.todense()
+    elif is_cupy_array(a):
+        a = a.get()
+    elif is_dask_array(a):
+        a = a.compute()
+    np.testing.assert_array_equal(a, b)
```

Comment:
This feels like a problem someone else already dealt with elsewhere.
Did I reinvent the wheel?